emloop_tensorflow/hooks/decay_lr.py

```diff
@@ -123,8 +123,7 @@ class DecayLROnPlateau(el.hooks.OnPlateau, DecayLR):
     """
 
     def __init__(self, decay_value: float=0.1, **kwargs):
-        el.hooks.OnPlateau.__init__(self, **kwargs)
-        DecayLR.__init__(self, decay_value=decay_value, **kwargs)
+        super().__init__(decay_value=decay_value, **kwargs)
```

Comment:
this isn't correct - see an example

```ipython
In [5]: class A:
   ...:     def __init__(self): print('a')
   ...: 
   ...: 

In [6]: class B:
   ...:     def __init__(self): print('b')
   ...: 
   ...: 

In [7]: class C(A, B):
   ...:     def __init__(self): super().__init__()
   ...: 
   ...: 

In [8]: c = C()
a
```

`super` seems to take just one of the parents i.e. you never initialize `OnPlateau` parent which is why the warning is gone.

I believe the correct way of doing this is 
```
def __init__(self, decay_value: float=0.1, long_term=0, short_term=0, **kwargs):
    el.hooks.OnPlateau.__init__(self, long_term, short_term, **kwargs)
    DecayLR.__init__(self, decay_value=decay_value, **kwargs)
    ...
```

Passing `**kwargs` to both constructors seems a bit off, though.

Reply:
it does get called, if I put prints to both inits, they get printed, in all tests with this hook
```
.test_arg_forward
on_plateau init
decay_lr init
.test_call_forward
on_plateau init
decay_lr init
.test_wait
on_plateau init
decay_lr init
.test_variables_dont_raise_warning
on_plateau init
decay_lr init
```
I found this solution on stackoverflow, because as it was written before, DecayLR was called twice, and somebody there was struggling with exactly same problem and their recommended solution was to use super().__init__() which apparently works